### ISSUE TITLE
Step: Add basic manifest.json using "legacy" option initially

### ISSUE THREAD

**Issue by cleidigh** [cleidigh/EditEmailSubject-MX#6]
Nothing can proceed without a manifest file for 68+
- Create new branch - name?? v3.0.0b1-manifest
- Begin with the "hybrid legacy option" so we can execute under 60 for overlays/testing
- Skip optional things like localization for now?
- Get baseline errors
- Reference first step in DTN 68 conversion steps

**Comment by opto** [comment]
why executable uner 60 - doeesn't that make code more intransparent?

**Comment by cleidigh** [comment]
@opto 
Your comment goes to a broader question we need to settle on:

What sequence in terms of compatibility to want to do.  I put 60 only because it works now, I am only suggesting an intermediary step not that we make it compatible with 60 for the final work.
I think it is a good group question about how we do changes for the intermediary work.
For instance to we also want to make one steps compatibility with 68 or proceed directly to
something with as many MX APIs?

**Comment by opto** [comment]
we will be using 68 for another year so it should also be compatible with68

**Comment by cleidigh** [comment]
@opto 
Absolutely.  I probably should have put a clear range:
68.* - 7x
At some point we will probably have to fix which 7x will be the top version.

**Comment by cleidigh** [comment]
I have not heard any comments on localization for the manifest.
I have changed my mind as it's only two strings might just as well show it?

**Comment by jobisoft** [comment]
Didn’t we decide that already? We should not circle back to questions we have already settled upon. From my understanding we are going for a pure WebExt without legacy flag. This will work in 68 and upwards. It could be, that an API is only added later and thus not avail in 68, but we can backport those via experiments.

**Comment by cleidigh** [comment]
I pushed a branch with a base manifest , legacy option just to load overlays for the first step.
no localization, popup should work but does not .  I have posted a question on this because it also stopped my hello world example from working for some reason.
Tomorrow we should figure out who wants to do what next.

**Comment by cleidigh** [comment]
okay, I was beginning to think I'm going crazy, the pop-up icon was hidden
it does work, so this is a pretty good starting point

![image](https://user-images.githubusercontent.com/25272315/71223808-fc3b1c00-22a2-11ea-8351-c96f79de87e1.png)

**Comment by jobisoft** [comment]
Can we consider the [TB68 branch](https://github.com/cleidigh/EditEmailSubject-MX/commits/TB68) to solve this issue? 

Can this issue then be closed or do we want it to remain open for visibility?

**Comment by jobisoft** [comment]
The TB68 branch has been merged into master and this step is now completed. I suggested to use the release page for detailed curated logging of the update steps for now. We still need to discuss, if the release page is a good place for this, or if we should use a wiki page or an issue. The discussion for that is here: https://github.com/cleidigh/EditEmailSubject-MX/issues/5

As this issue has been solved, it can be closed. If there is any information from here missing in the final update log, we can either link to this issue or clone the relevant information.
